git-coverage

```diff
@@ -792,8 +809,15 @@ def main(argv):
             patches_by_filename[filename] = []
         patches_by_filename[filename].append(patch)
 
+    omit_patterns = get_omit_patterns()
     # Now go through and calculate coverage
     for (filename, patches) in patches_by_filename.items():
+        if any([fnmatch.fnmatch(filename, omit_pattern)
+            for omit_pattern in omit_patterns]):
+            # print skipped file in yellow
+            print "\033[93m%s is skipped\033[0m" % filename
```

Comment:
We shouldn't use ascii colouring here - much of the output is going to be in log files, so using colouring makes the text harder to read.
You could have "SKIPPING FILE: %s" to highlight it